tests/run-pass/other_constants.rs

```diff
@@ -10,3 +10,10 @@ pub static A: bool = true;
 pub static B: bool = false;
 pub static C: char = 'A';
 pub static D: &str = "foo";
+
+pub fn main() {
+    debug_assert!(A == true);
```

Comment:
Wondering: why use `debug_assert` here rather than `assert`? Would we ever want to run the tests with the asserts compiled away?